docs/SMARTonFHIR.md

```diff
@@ -0,0 +1,44 @@
+[SMART on FHIR](https://docs.smarthealthit.org/) is a set of open specifications to integrate partner applications with FHIR servers and electronic medical records systems that have FHIR interfaces. 
+
+There is a tutorial on the managed service documentation that describes how to use the proxy to enable SMART on FHIR applications with the Azure API for FHIR. You can follow the [steps](https://docs.microsoft.com/azure/healthcare-apis/use-smart-on-fhir-proxy) for the open-source FHIR Server as well with a few changes that are highlighed below.
+
```

Comment:
Do I need to do all the steps and then apply steps in this tutorial, or I need to mix them? And in general it's a bit mentally hard to jump between two open browser pages back and forth.

Reply:
I updated this. Hopefully this is clearer!